test/timezone/test_mathn_compatibility.rb

```diff
@@ -2,12 +2,18 @@
 
 require 'timezone'
 require 'minitest/autorun'
-require 'mathn'
 
-class TestTimezone < ::Minitest::Test
-  parallelize_me!
+begin
+  require 'mathn'
 
-  def test_lookup_mathn_compatibility
-    Timezone['America/Regina'].utc_offset
+  class TestTimezone < ::Minitest::Test
+    parallelize_me!
+
+    def test_lookup_mathn_compatibility
+      Timezone['America/Regina'].utc_offset
+    end
   end
+rescue LoadError => e
+  raise e until e.path == 'mathn'
```

Comment:
Can you explain the use of `until` here? It seems unconventional. Why not use `raise unless e.path == 'mathn'`?

Reply:
Yes, sorry, my bad. Fixed.